deploy/charts/cert-manager/values.yaml

```diff
@@ -74,6 +74,11 @@ image:
 # used. This namespace will not be automatically created by the Helm chart.
 clusterResourceNamespace: ""
 
+# This namespace allows you to define where the services will be installed into
+# if not set then they will use the namespace of the release
+# This is helpful when installing cert manager as a chart dependency (sub chart)
+namespace: ""
```

Comment:
Please link to this Helm issue which nicely describes the problem and has links to other projects and their helm charts which are also using this technique to  work around this problem:
  * https://github.com/helm/helm/issues/5358

Reply:
Added